rpm/tarantool.spec

```diff
@@ -57,18 +57,22 @@ Requires(preun): chkconfig
 Requires(preun): initscripts
 %endif
 
-%if 0%{?rhel} >= 8
-# gh-4611: Disable backtraces on CentOS 8 by default due to lack
-# of libunwind package in the base system.
+# Enable backtraces everywhere, except AARCH64 and ancient GCC versions, which
+# lack compiler features required for backtrace.
+%define __cc "%{getenv:CC}"
+%if %{__cc} == ""
+%define __cc "cc"
+%endif
+%if "%(printf '%%s\n' "5.3.0" "$(%{__cc} -dumpfullversion -dumpversion)" | sort -V | head -n1)" != "5.3.0"
```

Comment:
So the resulting package may have different feature set depending of what is installed on a build machine? It should not be so, I think.

I guess it is workaround for the CI workflow with gcc 4. I was initially against implementing it via the RPM package build. It seems, now we meet consequences.